src/main/java/heronarts/lx/LXComponent.java

```diff
@@ -884,11 +885,19 @@ protected LXComponent addParameter(String path, LXParameter parameter) {
     }
     if (parameter instanceof ColorParameter) {
       // NOTE: order is important, brightness must be saved/loaded first
+      // TODO: Why?
       ColorParameter colorParameter = (ColorParameter) parameter;
       addParameter(path + "/" + ColorParameter.PATH_BRIGHTNESS, colorParameter.brightness);
       addParameter(path + "/" + ColorParameter.PATH_SATURATION, colorParameter.saturation);
       addParameter(path + "/" + ColorParameter.PATH_HUE, colorParameter.hue);
     }
+    if (parameter instanceof LinkableColorParameter) {
+      LinkableColorParameter linkableColorParameter = (LinkableColorParameter) parameter;
+      addParameter(path + "/" + LinkableColorParameter.PATH_SWATCH_NUMBER,
```

Comment:
I don't love referencing swatch number, this will break in weird ways if a swatch is deleted, or swatches are reordered. I'm not really sure that it should be possible to reference saved swatches vs. referencing the active swatch.

I had anticipated this desire before, as it particularly seems tempting to want two active swatches in a crossfader situation (e.g. swatch A and B for either side). But it's a potentially slippery slope to active-swatches-upon-swatches... I'll mull this further.

Referencing an active swatch color position by index def feels good though.